bluesea_node:
  ros__parameters:
  #ROS2#
    frame_id: "map"
    topic: "scan"
  #DATA#
    dev_id: -1
    raw_bytes: 3
    output_360: true
    output_scan: true
    output_cloud: false
    with_angle_filter: false
    min_angle: -3.14159
    max_angle: 3.14159
    mask1: "0,3.14"
    mask2: "-3,-2"
    hard_resample: true
    soft_resample: false
    with_checksum: true
    inverted: false
    reversed: false
    min_dist: 0.01
    max_dist: 50.0
    error_circle: 3
    error_scale: 0.9
  #CONNECT#
    type: "udp"
    lidar_ip: "192.168.0.88"
    lidar_port: 6543
    local_port: 6668
    group_ip: "224.1.2.3"
    #FITTER#
    filter_open: false  #滤波生效开关
    max_range: 20.0 #滤波生效的最大距离
    min_range: 0.5 #滤波生效的最小距离
    max_range_difference: 0.1 #离异点判断的物理范围
    filter_window: 1 #判断离异点下标的范围
  #GET#
    uuid:  -1
  #SET#
    resample_res: -1.0
    with_confidence: -1
    with_smooth: -1
    with_deshadow: -1
    alarm_msg: -1
    rpm: -1
    direction:  -1
    ats: -1
